lib/rubocop/cop/rspec/expect_change.rb

```diff
@@ -64,9 +66,11 @@ def on_block(node)
           return unless style == :method_call
```

Comment:
Since you've touched handling of both styles, `block` and `method_call`, please consider adding examples under `context 'with EnforcedStyle `method_call`' do` as well.

Reply:
We added examples under context with EnforcedStyle method_call.